package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "merlin",
   "version": "1.0.0",
-  "description": "",
+  "description": "merlin and the bg's",
```

Comment:
do git revert <commit hash> n then only check in relevant files, also tests need to be refactored too cuz that's where the actions are actually built